bot/src/notify.ts

```diff
@@ -16,6 +16,15 @@ const generateNotificationTextSurplus = function (auction: SurplusAuctionActive)
     )} DAI just started. Follow the link to participate: ${url}`;
 };
 
+const generateNotificationTextDebt = function (auction: DebtAuctionActive): string {
+    const url = `${process.env.FRONTEND_ORIGIN}/debt/?network=${auction.network}&auction=${auction.id}`;
+    return `Debt auction with fixed bid amount of ${auction.bidAmountDai.toFixed(
```

Comment:
Why do we not use the new `formatToAutomaticDecimalPointsString` function you introduced? The same also goes for the SurplusAuction Notification message. 

Maybe we want to change both in this PR?